src/main/java/org/folio/circulation/domain/policy/LoanPolicyRepository.java

```diff
@@ -21,10 +26,12 @@
 public class LoanPolicyRepository extends CirculationPolicyRepository<LoanPolicy> {
 
   private final CollectionResourceClient fixedDueDateSchedulesStorageClient;
+  private final CollectionResourceClient loanPoliciesClient;
 
   public LoanPolicyRepository(Clients clients) {
     super(clients.circulationLoanRules(), clients.loanPoliciesStorage());
     this.fixedDueDateSchedulesStorageClient = clients.fixedDueDateSchedules();
+    this.loanPoliciesClient = clients.loanPoliciesStorage();
```

Comment:
Why has a new field been introduced over using the existing `policyStorageClient`?

Reply:
Resolved